mbuild/formats/hoomd_snapshot.py

```diff
@@ -104,11 +103,11 @@ def to_hoomdsnapshot(
     structure : parmed.Structure
         ParmEd Structure object
     ref_distance : float, optional, default=1.0
-        Reference distance for conversion to reduced units
+        Reference distance for unit conversion (from Angstrom)
```

Comment:
What happens if ref_distance is used and autoscale is turned on? Might need a note somewhere about that case.